### PR TITLE
Ivan/dataset searcher

### PR DESCRIPTION
**Pull request by Josmorsot**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "@carto/carto.js": "^4.2.1",
     "@carto/toolkit-core": "0.0.1-rc.18",
     "@carto/toolkit-custom-storage": "0.0.1-rc.18",
-    "@carto/viewer": "CartoDB/viewer#v1.0.2",
+    "@carto/viewer": "CartoDB/viewer#refinement",
```
Comment (padawannn) [inline package.json:25]: change the viewer version

File: script/ci/cloudbuild-tests-pr-pg12.yaml [hunk 2]
```diff
@@ -88,11 +88,11 @@ steps:
   timeout: 900s
 
 - name: gcr.io/cloud-builders/docker
-  args: ['exec', '-i', 'builder_1', 'bash', '-c', '/cartodb/runParallelTests.sh 24' ]
-  timeout: 3000s
+  args: ['exec', '-i', 'builder_1', 'bash', '-c', '/cartodb/runParallelTests.sh 12' ]
+  timeout: 2700s
```
Comment (padawannn) [inline script/ci/cloudbuild-tests-pr-pg12.yaml:92]: why this?

**Review by padawannn — CHANGES_REQUESTED** [review]

File: script/ci/cloudbuild-tests-pr-pg12.yaml (hunk 2 above)
Comment (Josmorsot) [reply script/ci/cloudbuild-tests-pr-pg12.yaml:92]: It has been added by @alberhander, to solve a problem with CI

**Review by padawannn — APPROVED** [review]

**Comment by rafatower** [comment]
:robot: please make sure all the checks pass before merging @Josmorsot

**Pull request merged**